src/components/ui/Checkbox/checkbox.scss

```diff
@@ -1,26 +1,65 @@
 @import "src/styles/scaffold";
 
-[data-store-checkbox] {
+[data-fs-checkbox] {
+  // --------------------------------------------------------
+  // Design Tokens for Checkbox
+  // --------------------------------------------------------
+
+  // Default properties
+  --fs-checkbox-width                        : 1.25rem;
+  --fs-checkbox-height                       : var(--fs-checkbox-width);
+
+  --fs-checkbox-border-width                 : var(--fs-border-width);
+  --fs-checkbox-border-color                 : var(--fs-border-color);
+  --fs-checkbox-border-radius                : var(--fs-border-radius);
+
+  --fs-checkbox-transition                   : border var(--fs-transition-timing) var(--fs-transition-function), background-color var(--fs-transition-timing) var(--fs-transition-function), box-shadow var(--fs-transition-timing) var(--fs-transition-function);
+
+  --fs-checkbox-bkg-color-hover              : var(--fs-color-primary-bkg-light);
+  --fs-checkbox-border-color-hover           : var(--fs-color-text);
```

Comment:
`--fs-checkbox-border-color-hover` should be near `--fs-checkbox-border-...` definitions

Reply:
done [here](https://github.com/vtex-sites/gatsby.store/pull/61/commits/7557dc515c07971f504aaa53e1e908ac225280d7)